src/runtime/converter.cs

```diff
@@ -133,6 +134,22 @@ internal static IntPtr ToPython(object value, Type type)
                 return result;
             }
 
+            if (value is IList && value.GetType().IsGenericType)
+            {
+                using (var resultlist = new PyList())
+                {
+                    foreach (object o in (IEnumerable)value)
```

Comment:
No, I meant this place:

https://github.com/pythonnet/pythonnet/blob/423407a4e7ff783d8f4d9498eaa51bf8a12382ce/src/runtime/converter.cs#L208